layouts/partials/head.html

```diff
@@ -27,18 +27,11 @@
 <meta property="og:image" content="{{ index . 0 | replaceRE ".*src=\"(.+?)\".*" "$1" }}" />
 {{ end }}
 {{ end }}
-<script
-src="https://code.jquery.com/jquery-3.2.1.min.js"
-integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-crossorigin="anonymous"></script>
-<script
-src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
-integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
-crossorigin="anonymous"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.js"></script>
+<script src="{{ "js/jquery-3.2.1.min.js" | relURL }}"></script>
+<script src="{{ "js/jquery-ui.min.js" | relURL }}"></script>
```

Comment:
@lucperkins Can you rename the js files to include the version number? Specifically, follow the `js/jquery-3.2.1.min.js` example. It would be helpful for us to know which version of each component we are using.